ballerina/tests/execute-basic-test.bal

```diff
@@ -260,9 +260,9 @@ function testInsertWithStringAndSelectTable() returns error? {
     StringData expectedInsertRow = {
         id: 25,
         varchar_type: "str1",
-        charmax_type: "str2",
+        charmax_type: "str2      ",
```

Comment:
is it intentional

Reply:
Yes, With h2 2x...  when the varchar max character is defined the rest of the values are filled with spaces.